### PR TITLE
feat: add f16 squawk docs

### PR DESCRIPTION
**Pull request by M3ales**
Video provided by `@xVersify#5761` (Discord).

May need to do some clearer close ups of the IFF panel itself.

### CODE REVIEW

File: docs/pilot-guide/dcs_mode_3_squawk.md [hunk 1]
```diff
@@ -41,16 +41,45 @@ Their mode 3 transponder label will also read `EMER` if `7700` or `RDO` if `7600
...
+## F16C Viper
 
-!!! todo Todo
-	This section has not been completed yet, but you can assist by [adding to the thread on discord](https://discord.com/channels/1099799536416862330/1102555371404525640), or by submitting a pull request to update this section.
+### Mode 3 Transponder
+
+![F16 Mode 3 Setup Animation](../assets/pilot-guide/transponder/f16_iff_mode_3_setup.webp)
+
+!!! note
+	The viper does not differentiate between mode 1 and 3 within the cockpit. You will use both the mode 1 and 3 section to input your mode 3 squawk (4 digits).
```
Comment (QuantifyGG) [inline docs/pilot-guide/dcs_mode_3_squawk.md]: Viper should be capitalised

**Review by QuantifyGG — APPROVED** [review]
Looks good, thanks. One little typo, feel free to merge once it's fixed !

**Comment by M3ales** [comment]
@QuantifyGG no write access to repo so can't merge, need you to do it.

**Comment by QuantifyGG** [comment]
Oops, thanks

**Pull request merged**